config/enabled.yml

```diff
@@ -1645,6 +1645,11 @@ Rails/HasAndBelongsToMany:
   StyleGuide: 'https://github.com/bbatsov/rails-style-guide#has-many-through'
   Enabled: true
 
+Rails/HasManyDependent:
```

Comment:
The name of the cop is a bit misleading (because it's not clear it handles `has_one` as well).